### ISSUE TITLE
Incorrect behavior in SSR mode

### ISSUE THREAD

**Issue by ilyaskorik** [vuex-orm/vuex-orm#242]
In SSR mode, we recover store from saved data:

```
if (window.__INITIAL_STATE__) {
  store.replaceState(window.__INITIAL_STATE__)
}
```

But Models are not restored when the store is replaced in SSR mode, so accessors and mutators are not available.

**Comment by kiaking** [comment]
Hi! thank you for the feedback! Interesting. I guess `replaceState` function is doing deep copy of the state? Hence model instances are becoming plain object maybe?

Is `window.__INITIAL_STATE__` list of model instances?

**Comment by kiaking** [comment]
This issue should been solved by Vuex ORM 0.28.0! 🔥  Thank you so much for the report!
https://github.com/vuex-orm/vuex-orm/releases/tag/v0.28.0
